### ISSUE TITLE
Save state of tabs more frequently

### ISSUE THREAD

**Issue by Gabriel-p** [elementary/files#1744]
## Problem
When the app closes unexpectedly (crash or system reset) the opened tabs are lost.

## Proposal
Save the state of the opened tabs more frequently?

**Comment by cassidyjames** [comment]
Yeah, ideally their state should be saved when they change, not on close. The argument I've head against this is that if a tab's state is causing a crash, it may become impossible to recover without manually editing GSettings (or handling that case in Files as well).

**Comment by jeremypw** [comment]
Closing as the tabs are now saved whenever one is successfully loaded.
